discovery/kubernetes_api_discovery_test.go

```diff
@@ -54,7 +54,13 @@ func (m *mockK8sDiscoveryCommand) GetServices() ([]byte, error) {
 	               {
 	                  "port" : 10007,
 	                  "protocol" : "TCP",
-	                  "targetPort" : 8088
+	                  "targetPort" : 8088,
+					  "nodePort": 38088
+	               },
+	               {
+	                  "port" : 10008,
+	                  "protocol" : "TCP",
+	                  "targetPort" : 8089
```

Comment:
Adding this to make sure it gets filtered and doesn't show up in the later tests